crawler/file_processing.py

```diff
@@ -386,6 +395,14 @@ def set_state_for_file(self) -> CentreFileState:
         elif self.checksum_match(SUCCESSES_DIR):
             self.file_state = CentreFileState.FILE_PROCESSED_WITH_SUCCESS
 
+        # check for this being an unconsolidated samples file where the centre doesn't support those
+        elif (
+                CENTRES_KEY_SKIP_UNCONSOLIDATED_FILES in centre and
```

Comment:
Could we use `centre.get(CENTRES_KEY_SKIP_UNCONSOLIDATED_FILES, False) and self.is_unconsolidated_surveillance_file() ` instead?